### PR TITLE
(WSR 8) - <Label/> - remove component

### PR DESCRIPTION
**Pull request by EgoziE**
Removing label component
(next PR removing wix-ui-backoffice dependency)

### CODE REVIEW

**Comment by EgoziE** [comment]
Checked - Failing tests are unrelated.

**Comment by EgoziE** [comment]
> Don't forget to add this to the migration guide

done

File: MIGRATION.md [hunk 1]
```diff
@@ -544,6 +545,10 @@ Props:
 - removed `disableClickOutsideWhenClosed`, this functionality is now permanent <br/>
   Just remove this prop, no other change required.
 
+## \<Label/>
+
+This component was deleted, use native label instead.
```
Comment (shlomitc) [inline MIGRATION.md]: I don't think that's the appropriate solution.
We should advise on using either:
1. `<FormField/>`
2. `<Text tagName="label">`

Comment (EgoziE) [reply MIGRATION.md]: Thanks! I'll write a better example.

**Pull request merged**